pytorch_lightning/accelerators/ddp_accelerator.py

```diff
@@ -230,12 +232,15 @@ def ddp_train(self, process_idx, model):
         # try to init for 20 times at max in case ports are taken
         # where to store ip_table
         model.trainer = self.trainer
-        self.init_ddp_connection(
+        should_return = self.init_ddp_connection(
```

Comment:
Can we potentially separate this behaviour from the init ddp connection? Like `ddp_plugin.await_process()` or something?